apps/fluent-tester/src/E2E/common/consts.ts

```diff
@@ -1,4 +1,4 @@
-/* Accessibility Role Constants - https://microsoft.github.io/windows-docs-rs/doc/bindings/Windows/Win32/UI/Accessibility/ */
+/* Accessibility Role Constants - https://microsoft.github.io/windows-docs-rs/doc/windows/Win32/UI/Accessibility/index.html */
```

Comment:
I'm not really sure what this link is for. Is it related to the accessibility testing api you used? If you're trying to show what these roles/control types are (or show the constants that are used), I'd recommend using the Microsoft Docs website: https://docs.microsoft.com/en-us/windows/win32/winauto/uiauto-controltypesoverview#current-ui-automation-control-types or https://docs.microsoft.com/en-us/windows/win32/winauto/uiauto-controltype-ids

Reply:
You're right. This is actually the old link I was using that's not valid anymore. I'll update it with one of the ones above Thanks!